### PR TITLE
Add docker with postgresql and named volumes

### PR DESCRIPTION
**Pull request by KangoV**
My first PR :) Be gentle.

### CODE REVIEW

**Comment by andreynering** [comment]
Trusted LGTM

File: content/doc/installation/with-docker.en-us.md [hunk 1]
```diff
@@ -35,6 +39,66 @@ docker run -d --name=gitea -p 10022:22 -p 10080:3000 -v /var/lib/gitea:/data git
...
+    -e POSTGRES_PASSWORD=<PASSWORD> \
+    -v gitea-db-data:/var/lib/postgresql/data \
+    -p 5432:5432 \
+    postgres:alpine
+```
+
+Now the database is up and running, we need to configure it. Make sure you remember the password for when Gitea starts.
+
+```
+$ docker exec -it gitea-db psql -U postgres:alpine
```
Comment (KangoV) [inline content/doc/installation/with-docker.en-us.md]: Ahh that won't do. User should be `postgres`.

**Comment by tboerger** [comment]
Maybe it should be better to create a docker compose directory on the examples repo and link to that from the docs?

**Comment by KangoV** [comment]
I'm just learning docker compose now. As soon as I've got it sorted I'll do another PR :)

**Comment by pgaskin** [comment]
LGTM

**Comment by lunny** [comment]
let L-G-T-M work

**Review by appleboy — APPROVED** [review]

**Comment by lunny** [comment]
let L-G-T-M work

**Comment by lunny** [comment]
let L-G-T-M work

**Comment by tboerger** [comment]
As I said above this is better within the examples repository -.-

**Comment by metalmatze** [comment]
I thought about creating such a examples directory too. I know deployed gitea in docker swarm with docker-compose v3. I could easily provide my file and create a simpler docker-compose (v1) too. Somebody was already on Kubernetes as well.

**Pull request merged**